starlette/testclient.py

```diff
@@ -381,13 +381,11 @@ def receive_json(self, mode: str = "text") -> typing.Any:
 
 class TestClient(requests.Session):
     __test__ = False  # For pytest to not discover this up.
-
-    #: These options are passed to `anyio.start_blocking_portal()`
+    #: These are the default options for the constructor arguments
     async_backend: typing.Dict[str, typing.Any] = {
```

Comment:
Do we need this class-level declaration still? Class vars always confuse me, but this means somebody can do:
```python
TestClient.async_backend["backend"] = "trio"
```
to default all future `TestClient` instances to use the Trio backend, right?

I know removing this would be slightly changing the API, however in the code examples we gave we only modified instance attributes and that usage should stay the same. What's your thinking around this?

Reply:
I've removed it in my followup PR: https://github.com/encode/starlette/pull/1210